pkg/convert/xliff2.go

```diff
@@ -1,17 +1,38 @@
 package convert
 
 import (
+	"bytes"
 	"encoding/xml"
+	"errors"
 	"fmt"
+	"io"
+	"strconv"
+	"strings"
 
+	"go.expect.digital/mf2"
 	"go.expect.digital/translate/pkg/model"
 	"golang.org/x/text/language"
 )
 
 // TODO: For now we can only import XLIFF 2.0 files, export is not working correctly yet.
 
-// XLIFF 2 Specification: https://docs.oasis-open.org/xliff/xliff-core/v2.0/os/xliff-core-v2.0-os.html
-// XLIFF 2 Example: https://localizely.com/xliff-file/?tab=xliff-20
+// This implementation follows OASIS Standard for XLIFF 2.0 specification (last updated 2014-08-05).
+// XLIFF 2.0 Specification: https://docs.oasis-open.org/xliff/xliff-core/v2.0/os/xliff-core-v2.0-os.html
+
+// NOTE: Xliff 2.0 has no unified standard for storing fuzzy values, plurals, gender specific text.
+
+// List of elements found in source/target:
+// For more details: https://docs.oasis-open.org/xliff/xliff-core/v2.0/os/xliff-core-v2.0-os.html#source
+const (
+	cp  = "cp"  // Unicode character that is invalid in XML
+	ph  = "ph"  // standalone code of the original format
+	pc  = "pc"  // well-formed spanning original code
+	sc  = "sc"  // start of a spanning original code
+	ec  = "ec"  // end of a spanning original code
+	mrk = "mrk" // annotation pertaining to the marked span
+	sm  = "sm"  // start marker of an annotation
+	em  = "em"  // end marker of an annotation
+)
 
 type xliff2 struct {
```

Comment:
i think structs fields have to be unexported

Reply:
aren't they exported already?

Reply:
Well, for Unmarshal to struct we need exported.

Reply:
sure, forgot :)